src/Scriban/Runtime/IItemAccessor.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Alexandre Mutel. All rights reserved.
+// Licensed under the BSD-Clause 2 license.
+// See license.txt file in the project root for full license information.
+#nullable disable
+
+using System;
+using Scriban.Parsing;
+
+namespace Scriban.Runtime
+{
+    public interface IItemAccessor
```

Comment:
Maybe add a comment to explain that it is for accessing indexers on object.

Reply:
Updated